packages/flutter/lib/src/material/stepper.dart

```diff
@@ -326,7 +374,10 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
     }
   }
 
-  Widget _buildVerticalControls() {
+  Widget _buildVerticalControls(ControlsWidgetBuilder builder) {
+    if(builder != null)
```

Comment:
Instead reference widget.controlsBuilder directly and don't pass it around everywhere